includes/class-wc-payments-woopay-direct-checkout.php

```diff
@@ -89,7 +89,7 @@ public function scripts() {
 			return;
 		}
 
-		if ( ! $this->woopay_utilities->should_enable_woopay_on_cart_or_checkout() ) {
```

Comment:
As the mini cart shows up in all pages should we also update this call? https://github.com/Automattic/woocommerce-payments/blob/4429d05714e2d1b0b1627529e4c66a40bfd8e049/includes/class-wc-payments-checkout.php#L202

If so what about removing the old method completely?

Reply:
Fixed on https://github.com/Automattic/woocommerce-payments/pull/10917/commits/c75d3e2839871116b8154f3cb5041f6973922fb6.